src/d3d9/d3d9_device.cpp

```diff
@@ -723,39 +736,67 @@ namespace dxvk {
       return D3DERR_INVALIDCALL;
 
     const Rc<DxvkImage> dstImage  = dstTexInfo->GetImage();
-      
+    const DxvkFormatInfo* formatInfo = imageFormatInfo(dstTexInfo->GetFormatMapping().FormatColor);
     uint32_t mipLevels   = std::min(srcTexInfo->Desc()->MipLevels, dstTexInfo->Desc()->MipLevels);
     uint32_t arraySlices = std::min(srcTexInfo->Desc()->ArraySize, dstTexInfo->Desc()->ArraySize);
+
+    if (unlikely(srcTexInfo->IsAutomaticMip() && !dstTexInfo->IsAutomaticMip()))
```

Comment:
I'm scared of this

Reply:
Docs seem to say this is correct so I guess we can go with it for now...

Docs have lied in the past however :frog:

Reply:
Write a test then.

Reply:
FWIW, Nine does the same thing.
https://gitlab.freedesktop.org/mesa/mesa/-/blob/master/src/gallium/frontends/nine/device9.c#L1460